common/libs/validators.ts

```diff
@@ -59,6 +59,12 @@ export function isValidBTCAddress(address: string): boolean {
   return WalletAddressValidator.validate(address, 'BTC');
 }
 
+export function isValidXMRAddress(address: string): boolean {
```

Comment:
This is a very naive checker, but our current libraries don't support Monero verification. I'm open to suggestions for how to handle this, whether it's adding a different library or anything else.